pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImplTest.java

```diff
@@ -785,6 +787,49 @@ public void testRemoveDeadBrokerTimeAverageData() throws Exception {
         assertEquals(data.size(), 1);
     }
 
+    @Test
+    public void testBundleDataDefaultValue() throws Exception {
+        final String tenant = "test";
+        final String cluster = "test";
+        String namespace = tenant + "/" + cluster + "/" + "test";
```

Comment:
This test sets the default bundle resource quota by Pulsar Admin V1 API, could you also add a case for Pulsar Admin V1 API?

Reply:
Done @poorbarcode